### PR TITLE
[15.0][MIG] helpdesk_mgmt_ticket_tree_recent

### PR DESCRIPTION
**Pull request by adriresu**

### CODE REVIEW

File: helpdesk_mgmt_ticket_tree_recent/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "summary": """
+        Adds group that sorts tickets on tree view by last update date.
+    """,
+    "version": "15.0.1.0.0",
+    "license": "LGPL-3",
+    "category": "After-Sales",
+    "author": "Solvos",
+    "website": "https://github.com/solvosci/slv-helpdesk",
+    "depends": [
+        "helpdesk_mgmt_timesheet",
```
Comment (dalonsod) [inline helpdesk_mgmt_ticket_tree_recent/__manifest__.py]: Notice that actually this dependency is unnecessary, try this:
```suggestion
        "helpdesk_mgmt",
```

File: helpdesk_mgmt_ticket_tree_recent/views/helpdesk_ticket_views.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="helpdesk_ticket_view_tree_write_date" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.view.tree (write_date)</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='create_date']" position="after">
+                <field name="write_date" />
```
Comment (dalonsod) [inline helpdesk_mgmt_ticket_tree_recent/views/helpdesk_ticket_views.xml]: Make `write_date` behavior the same as v15 for `create_date`, try this (see https://github.com/OCA/helpdesk/blob/77cdcc650300b418cfa88d183713afa6df989024/helpdesk_mgmt/views/helpdesk_ticket_views.xml#L237-L241):
```suggestion
                <field name="write_date" widget="remaining_days" optional="show" />
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments. When done, I'll merge it. Please add that changes to current migration commit, not creating another one.

File: helpdesk_mgmt_ticket_tree_recent/views/helpdesk_ticket_views.xml (hunk 2 above)
Comment (adriresu) [reply helpdesk_mgmt_ticket_tree_recent/views/helpdesk_ticket_views.xml]: Updated and squashed

File: helpdesk_mgmt_ticket_tree_recent/__manifest__.py (hunk 1 above)
Comment (adriresu) [reply helpdesk_mgmt_ticket_tree_recent/__manifest__.py]: Updated and squashed

**Review by dalonsod — APPROVED** [review]
LGTM

**Pull request merged**